modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationsPermissions.js

```diff
@@ -17,6 +19,9 @@ export default function PublicationsPermissions({
 	roles,
 	updatePermissionsURL,
 }) {
+	const [filteredRoles, setFilteredRoles] = useState(roles);
+	const [resultsKeywords, setResultsKeywords] = useState('');
```

Comment:
Why do we need a separate var for the results bar? Can't we just use the `searchTerms` in the results bar message?

Reply:
Because `searchTerms` will hold the current value of what's in the search box, even before we submit the search. I thought by adding resultsKeywords it would reflect an accurate value of what we are searching for at all times.  
If I use `searchTerms` in the search results bar, it may change while updating the input value in the search bar. I can test it and see if that happens.

Reply:
Yes, I needed `resultsKeywords` because the results bar was appearing just by typing in the search bar